<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Media</class>
 <widget class="QWidget" name="Media">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget 
{ 
   /*background-color: rgb(239, 245, 247); 
    color: black;*/
}

QPushButton,QMessageBox QPushButton
{
    font: 16px;
    font-weight: bold;
    color: white;
    background-color:DimGray;
    border-radius: 20px;

}

QPushButton:hover,QMessageBox QPushButton:hover
{
  background-color:rgb(23,157,186);/*???*/
    border-color:rgb(247,30,34);/*????*/
}
QLineEdit
{
    background-color:rgb(238,245,252);/*???*/
    border-style:solid;/*???? solid?? none??? inset/outset 3D??*/
    border-width:2px; /*??????*/
    border-radius:8px;/*????????*/
    border-color:rgb(86,213,220);/*????*/
    font-size:12pt;/*?? ????*/
    color: black;/*????*/
    padding:6px;/*??*/
}

QListWidget
{
    background-color:rgba(238,245,252,0.8);/*???*/
    border-style:solid;/*???? solid?? none??? inset/outset 3D??*/
    border-width:2px; /*??????*/
    border-radius:10px;/*????????*/
    border-color:DimGray;/*????*/;/*????*/

    color: DimGray;/*????*/
    padding:6px;/*??*/
}

QTextEdit {
    background-color:rgb(238,245,252);/*???*/
    border-style:solid;
    border-width:2px; /*??????*/
    border-radius:8px;/*????????*/
    border-color:rgb(86,213,220);/*????*/
    font-size:10pt;/*?? ????*/
    color: black;/*????*/
    padding:6px; /*??*/
}
QLabel {
    /*background-color:rgb(238,245,252);*/
    border-radius:8px;/*????????*/
    color:DimGray;/*????*/

}

#lrc_now{
    /*background-color:rgb(238,245,252);*/
    border-radius:8px;/*????????*/
	
	font: 20pt &quot;华文行楷&quot;;
    color:rgb(0, 170, 0);/*????*/;/*????*/
	/*text-decoration:underline;*/


}
#lrc_next{
    /*background-color:rgb(238,245,252);*/
    border-radius:8px;/*????????*/
	font: 16pt &quot;华文行楷&quot;;
	color:rgb(85, 170, 127);/*????*/;/*????*/
	/*text-decoration:underline;*/

}

QMessageBox {
	background-color: rgb(238,245,252); 
	font:12px;
	color: black;
}
QMessageBox QLabel { /* textLabel */
	color: rgba(23,157,186, 1);
	font:14px;
	background-color: rgba(238,245,252, 0.6);

}

/*QSlider::groove{
*background-color:#3e3e3e;
	border-radius:10px;}
*/

QSlider::handle:horizontal{
	background-color:#909090;
	border-radius:10px;
	width:10x;
	
}
QSlider::add-page{background-color:transparent;}
QSlider::sub-page{background-color:rgb(105,105,105);}</string>
  </property>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>400</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>添加歌曲</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>541</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Pause">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>||/▶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Forward">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>490</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Backward">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>51</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volumeup">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>500</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volumedown">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>550</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>V -</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Kufi Arabic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>400</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>移除歌曲</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>90</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 播放列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="timetext">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="medianow">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>561</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>正在播放:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lrc_now">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>当前歌词</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lrc_next">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>下句歌词</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timetext_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>540</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="next_song">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_song">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="list_button">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>540</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>三</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Add</zorder>
  <zorder>Pause</zorder>
  <zorder>Forward</zorder>
  <zorder>Backward</zorder>
  <zorder>Exit</zorder>
  <zorder>volumeup</zorder>
  <zorder>volumedown</zorder>
  <zorder>listWidget</zorder>
  <zorder>delete_2</zorder>
  <zorder>label_2</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>timetext</zorder>
  <zorder>medianow</zorder>
  <zorder>lrc_now</zorder>
  <zorder>lrc_next</zorder>
  <zorder>timetext_2</zorder>
  <zorder>next_song</zorder>
  <zorder>previous_song</zorder>
  <zorder>list_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
